presentation/src/main/java/bg/government/virusafe/app/registration/RegistrationFragment.kt

```diff
@@ -60,21 +64,54 @@ class RegistrationFragment :
 			}
 		}
 
-		binding.registrationTermsAndConditionsBtn.setOnClickListener {
+		binding.termsAndConditionsCheckBox.setOnCheckedChangeListener { _, isChecked ->
+			if (isChecked) {
+				setTermsAndConditionsColor(R.color.colorPrimary)
+			}
+		}
+
+		binding.registrationTermsAndConditionsTxt.setClickablePhrase(
+			fullText = "${viewModel.localizeString(I_AGREE_WITH_LBL)} " +
+					viewModel.localizeString(TERMS_N_CONDITIONS_SMALL_LBL),
```

Comment:
You don't need the '+' sign just put the TERMS_N_CONDITIONS_SMALL_LBL as well in the quotes.